### ISSUE TITLE
Workflow Question

### ISSUE THREAD

**Issue by achilleas101** [YingyiLiu/HAMS#35]
Dear @YingyiLiu,
I am trying to create the WAMIT file required for the [IEA 15-MW Monopile Wind Turbine](https://www.nrel.gov/docs/fy20osti/75698.pdf) using HAMS and I am facing some issues.
The steps I have taken so far are the following:

1. Created the monopile (from z=-75m to z=15m) and sea surface in Blender and exported both in one .stl file
2. Imported the .stl in BEMRosetta and exported it in .gdf format
3. Copied the .gdf file in the [MeshConverter](https://github.com/YingyiLiu/HAMS/tree/master/MeshConverter) folder of HAMS and changed the name of the file in the [Settings.ctrl](https://github.com/YingyiLiu/HAMS/blob/master/MeshConverter/Settings.ctrl) file. I ran RunWAMIT_MeshTran.bat and a HullMesh.pnl and WaterplaneMesh.pnl file were created
4. Copied the two resulting files of step 3 to the [Input folder](https://github.com/YingyiLiu/HAMS/tree/master/Compiling/Input) without making many changes in the ControlFile.in file except the water depth to 30m
5. Lastly I tried to run [HAMS_x64.exe](https://github.com/YingyiLiu/HAMS/blob/master/Bin/HAMS_x64.exe), but the following error occurred
![Screenshot 2024-07-09 093114](https://github.com/YingyiLiu/HAMS/assets/130864550/cac4617d-fbf3-4f15-a361-740b188eafcf)
How can I resolve this issue?

Additional questions:
i. Have I modeled the mesh correctly in step 1? Because the pile is cutting the seabed plane and it should be practically fixed.
ii. In step 3 I didn't change the center of gravity or rotation option. Should these values be computed from the wet part of 
the mesh only?

Any help would be greatly appreciated!
Best regards,
Achilleas

**Comment by gbarter** [comment]
Hello Achilleas,

Want to make sure you are aware of resources such as this [directory](https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/OpenFAST/IEA-15-240-RWT-UMaineSemi/HydroData) on the 15-MW repository.

Also, the NREL [WEIS](https://github.com/WISDEM/WEIS) tool works directly with HAMS (through [pyHAMS](https://github.com/WISDEM/pyHAMS/)) and will generate HAMS input files.  There is already an [example](https://github.com/WISDEM/WEIS/tree/main/examples) that uses the 15-MW turbine.

**Comment by achilleas101** [comment]
Hello gbarter,
Thank you for your fast reply! I am aware of this [directory](https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/OpenFAST/IEA-15-240-RWT-UMaineSemi/HydroData), but as far as I understand the files correspond to the UMaine semi submersible platform. Also thank you for your guidance with WEIS, I will give it a try!

Best regards,
Achilleas

**Comment by gbarter** [comment]
Yes, correct.  Sorry I didn't latch onto your specific need for the monopile and not the semisub.

**Comment by pelotonlopezmuro** [comment]
Hi @gbarter ! I have been exploring ways to generate HAMS input files, and in your post, you mentioned that WEIS can handle this process. I’d appreciate more details about the workflow:

- Does the process start from a CAD file (e.g., `.stl` or similar)?
- Where can I find the example involving the 15-MW turbine you mentioned?
- Does this process require any specific setup or configuration within WEIS or HAMS that I should be aware of?
- Are there any additional resources you would recommend to help understand this integration better?

Thank you in advance for your help!

**Comment by gbarter** [comment]
Hi there @pelotonlopezmuro 

[WEIS](https://github.com/WISDEM/WEIS) takes as an input [WIndIO](https://github.com/IEAWindTask37/windIO) inputs, which are yaml-based input files.  There is a workflow to start with CAD files if that is what you are looking for, but I'll have to refer you to someone else.

[Here](https://github.com/WISDEM/WEIS/tree/main/examples/06_IEA-15-240-RWT) is a WEIS example with the 15-MW, but you will likely need to start with the [WEIS documentation](https://weis.readthedocs.io/en/latest/) to get things running ([WISDEM documentation](https://wisdem.readthedocs.io/en/master/) might help too).
